src/main/webapp/availability.js

```diff
@@ -49,6 +49,6 @@ function updateAvailability() {
     firstSlot,
     lastSlot
   ].concat(Array.from(selectedSlots).map(s => s.getAttribute('data-utc')));
-  
   let request = new Request('/availability', {method:'PUT', body:JSON.stringify(utcTimestamps)});
+  fetch(request).then(unused => {loadAvailabilityTable(availabilityTableDiv(), browserTimezoneOffset())});
```

Comment:
Question: what is this doing? Updating the table with availabilities?
lf so we might experience some inconsistencies since we're not updating by key or anything, so eventual consistency might cause it show inaccurate results. :/

Reply:
This is reloading the table, which should be done after an update.

Reply:
It's within a promise, which should help somewhat. I'm not sure how much we should be compensating for datastore latency.